### PR TITLE
NXP-25763: Add nuxeo-ai

### PR DESCRIPTION
**Pull request by covolution**

### CODE REVIEW

File: marketplace.ini [hunk 1]
```diff
@@ -123,6 +123,9 @@ mp_to_upload = nuxeo-vision-marketplace/target/nuxeo-vision-marketplace-*.zip
 #depends on marketplace-dam
 other_versions = %(nuxeo-snapshot)s/%(nuxeo-tag)s/%(nuxeo-next_snapshot)s,%(marketplace-dam-snapshot)s/%(marketplace-dam-tag)s/%(marketplace-dam-next_snapshot)s
 
+[nuxeo-ai]
+mp_to_upload = **/nuxeo-ai*.zip
```
Comment (jcarsique) [inline marketplace.ini]: The pattern may contain simple shell-style wildcards a la fnmatch

https://docs.python.org/2/library/fnmatch.html

Pattern | Meaning
-- | --
`*` | matches everything
`?` | matches any single character
`[seq]` | matches any character in seq
`[!seq]` | matches any character not in seq

Comment (covolution) [reply marketplace.ini]: I need to match:
 `./nuxeo-ai-core-package/target/nuxeo-ai-core-2.0-SNAPSHOT.zip`
`./addons/nuxeo-ai-image-quality-package/target/nuxeo-ai-image-quality-package-2.0-SNAPSHOT.zip`
`./addons/nuxeo-ai-aws-package/target/nuxeo-ai-aws-package-2.0-SNAPSHOT.zip`

How do you handle sub-directories?  Is the match against the full path?

Comment (covolution) [reply marketplace.ini]: What language are you using to actually do the match? Python?  I looked for the code or tests but couldn't find it.  Starting with Python version 3.5, the glob module supports the "**" directive.  For older Python versions are you using `os.walk` to recursively walk a directory and `fnmatch.filter`?

Comment (jcarsique) [reply marketplace.ini]: It is still Python 2.7

https://github.com/nuxeo/nuxeo/blob/master/scripts/release_mp.py#L288

Maybe that an improvement is needed to circumvent the need of globbing.

Comment (covolution) [reply marketplace.ini]: I created NXP-26137 for the improvement.  For now lets go with `*/*/target/nuxeo-ai*.zip`

**Comment by jcarsique** [comment]
:+1:

**Comment by atchertchian** [comment]
:+1:

**Pull request merged**